### PR TITLE
docs: add installation notes to Atlantis home

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Our installation instructions are buried pretty deep in our IA - using Atlantis should be a key piece of info for contributors!

## Changes

Added packages overview and links to installation

### Added

- Added packages overview and links to installation

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -40,6 +40,28 @@ To start the [docz](https://www.docz.site/) development server:
 npm start
 ```
 
+### Installing packages
+
+Atlantis are installed and updated using [npm](https://www.npmjs.com/). This following list
```
Comment (eddysims) [inline docs/README.md:45]:
```suggestion
Atlantis packages are installed and updated using [npm](https://www.npmjs.com/). This following list
```

File: docs/README.md [hunk 2]
```diff
@@ -40,6 +40,28 @@ To start the [docz](https://www.docz.site/) development server:
...
+has installation links for each package:
+
+#### Design system
+
+These are the core packages you'll need to build with Atlantis:
+
+- [Components](https://atlantis.getjobber.com/packages/components#installation)
```
Comment (eddysims) [inline docs/README.md:52]: these links can be just `/packages/...`. This way they will work locally too

Comment (eddysims) [reply docs/README.md:52]: I would also say lets link these to the page and not the `installation` heading, This way the reader will understand the package as well

**Review by eddysims — CHANGES_REQUESTED** [review]

**Comment by chris-at-jobber** [comment]
@eddysims thanks! Back to you 🏓

**Review by eddysims — APPROVED** [review]

**Pull request merged**